01 - Community/Authors - Persons/kinabalu.md

```diff
@@ -0,0 +1,58 @@
+---
+aliases:
+- Andrew Lombardi
+tags: 
+- 
+publish: true
+---
+
+# Andrew Lombardi
+
+- GitHub: [kinabalu](https://github.com/kinabalu/) ^github
+
+%% - Discord: `@` ^discord %%
+
+- Website: <https://mysticcoders.com> ^website
+
+<!-- - [[Publish sites|Publish site]]: ^publish -->
+
+%% Feel free to add a bio below this comment %%
+
+
+## Author of
+
+%% Begin Hub: Released contributions %%
+### Plugins
+- [[obsidian-oura-plugin|Oura Ring]]
```

Comment:
When I view this file in Obsidian, it shows Oura Ring as a working link - so that means that there must have already been an author created for the Oura Ring plugin.

So I am puzzled as to why there is this whole extra new file, kinabalu.md

_Checking to see who the author of Oura Ring links to..._

Oh, it is a broken link in the published site... so for some reason, the plugin was created but not the author...

OK so I don't understand why, but it is **all good...** :-)

Reply:
that's just because I've been running the script in parts to make the PRs easier to review!